Lib/test/test_dict.py

```diff
@@ -37,6 +37,36 @@ def test_literal_constructor(self):
             dictliteral = '{' + ', '.join(formatted_items) + '}'
             self.assertEqual(eval(dictliteral), dict(items))
 
+    def test_pep_584(self):
+
+        a = {0: 0, 1: 1, 2: 1}
+        b = {1: 1, 2: 2, 3: 3}
+
+        c = a.copy()
+        c |= b
+
+        self.assertEqual(a | b, {0: 0, 1: 1, 2: 2, 3: 3})
```

Comment:
This assert ought to be placed before the definition of `c`.

Reply:
Hm, does this reasoning hold for the other assertions (below) as well? It seemed natural to group each pair of results since they evaluate the same.

Reply:
I guess what you're getting at is that we want to make sure the construction of `c` doesn't have side-effects on `a` and `b`?

Reply:
Not really, I just wanted a logical grouping of the tests -- but your grouping is more logical.